launch/launch/substitutions/python_expression.py

```diff
@@ -32,7 +33,8 @@ class PythonExpression(Substitution):
     Substitution that can access contextual local variables.
 
     The expression may contain Substitutions, but must return something that can
-    be converted to a string with `str()`.
+    be converted to a string with `str()`. It also may contain math symbols and
+    functions.
```

Comment:
nitpick: one sentence per line.
```suggestion
    be converted to a string with `str()`.
    It also may contain math symbols and functions.
```